pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
```

Comment:
Are you using `pnpm@8.3.1`? If not could you reinstall with it? Also make sure to use typescript version `5.0.4` as it the one currently used by Abitype.

Reply:
fixed in 24dd673